### ISSUE TITLE
Random token expiration date after setting it

### ISSUE THREAD

**Issue by carloscorrales** [ging/fiware-idm#148]
Using the provided docker image fiware/idm:7.7.0, we found a issue when using the IDM_OAUTH_ACC_LIFETIME environment variable. It doesn't use the value we pass, but a random number which increases everytime we ask for a new token, and then after some request, it goes back to the beginning.

For instance, we used `IDM_OAUTH_ACC_LIFETIME=31536000` (1 year) and, when asking for a new token we receive different values everytime we ask for a new token.

Are we missing any configuration? Is this a bug that should be fixed?

Thanks

**Comment by oscarvenini-perk** [comment]
I have also experienced problems with version 7.6.0 when setting IDM_OAUTH_ACC_LIFETIME to any number returns a token with some seconds of life.

Have you managed to make it work?

Thanks

**Comment by carloscorrales** [comment]
No, I wasn't able to fix it, it still returns a token with a random lifetime.

**Comment by Siedlerchr** [comment]
I also stumbled across this bug, still present in 7.9.2
